src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+import javax.sound.midi.SysexMessage;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    static Task[] tasks = new Task[100];
+    static int taskPosition = 0;
+    static String description;
+    static String by;
+    static int indexOfBy;
+
+    public static void storeTask(Task t) throws TodoException{
+        if (!t.description.isEmpty()){
+            tasks[taskPosition] = t;
+            taskPosition++;
+            System.out.println("Got it! I've added this task!");
+            System.out.println(t.description);
+            System.out.println(t.getStatusIcon() + " " + t.getDescription());
+            System.out.println("Now you have " + countArray(tasks) + " tasks!");
+
+        }
+        else {
+            throw new TodoException();
+        }
+
+    }
+
+    public static int countArray(Task[] tasks){
+        int counter = 0;
+        for (Task task:tasks){
+            if (task != null){
+                counter++;
+            }
+        }
+        return counter;
+    }
+
+    public static void markAsDone(int taskIndex){
+        System.out.println("Nice! I've marked this task as done: ");
+        tasks[taskIndex-1].isDone = true;
+        listArray(tasks);
+    }
+
+    public static void listArray(Task[] tasks){
+        int textNumber = 1;
+        for(Task t:tasks){
+            if(t != null){
+                System.out.println(textNumber + "." + t.getStatusIcon() +" " + t.getDescription());
+                textNumber++;
+            }
+        }
+    }
+
+    public static void main(String[] args) throws InvalidCommandException, InvalidDateException{
+        System.out.println("Hello! I'm 343 Guilty Spark! You may call me Spark!");
+        System.out.println("What can I do for you?");
+        String text = "hi";
+
+        while(!text.equalsIgnoreCase("Bye")){
+            Scanner in = new Scanner(System.in);
+            text = in.nextLine();
+            //Task t = new Task(text);
+            String[] arr = text.split(" "); //split command input into words
+            String command = arr[0];
+
+            switch (command){
+            case "deadline":
+            case "Deadline":
+            case "DEADLINE":
```

Comment:
These seem to refer to the same case, except that the string inputs are different. You could consider a separate way of handling such string inputs before executing the deadline block.